src/test/crimson/CMakeLists.txt

```diff
@@ -77,4 +77,4 @@ add_executable(unittest-seastar-errorator
 target_link_libraries(
   unittest-seastar-errorator
   crimson::gtest)
-
+add_ceph_unittest(unittest-seastar-errorator)
```

Comment:
Is it better to be renamed to `unittest_seastar_errorator`? To align with other crimson unit test names.

Reply:
i'd prefer using the foo-bar names, for two reasons

- less type. i need to press "shift" for input "_".
- more consistent with executable names like `ceph-conf`, `ceph-osd`.

we can change the others to be more consistent in another PR. but this is not in the scope of this change.

Reply:
OK, thanks. At least I know which way is preferred.

Reply:
#37955